README.md

```diff
@@ -382,12 +409,20 @@ theme={{
     dayTextAtIndex6: {
       color: 'blue'
     }
+  },
+  'stylesheet.day.basic': {
+    dayTextAtIndex0: {
+      color: 'red'
+    },
+    dayTextAtIndex6: {
+      color: 'blue'
+    }
   }
 }}
 ```
 
 <kbd>
-  <img height=50 src="https://github.com/wix-private/wix-react-native-calendar/blob/master/demo/day-header-style.png?raw=true">
+  <img height=50 src="https://github.com/wix-private/wix-react-native-calendar/blob/master/demo/day-header-and-days-style.png?raw=true">
```

Comment:
basically the only changes in README is from line 399 -> line 419

Reply:
Did you try pulling master to your branch and solve conflicts?